backend/danswer/connectors/confluence/connector.py

```diff
@@ -732,16 +640,18 @@ def _get_doc_batch(
                 continue
             page_text = parse_html_page(page_html, self.confluence_client)
 
-            files_in_used = get_used_attachments(page_html)
+            files_in_use = get_used_attachments(page_html)
             attachment_text, unused_page_attachments = self._fetch_attachments(
-                self.confluence_client, page_id, files_in_used
+                self.confluence_client, page_id, files_in_use
             )
             unused_attachments.extend(unused_page_attachments)
 
             page_text += "\n" + attachment_text if attachment_text else ""
             comments_text = self._fetch_comments(self.confluence_client, page_id)
             page_text += comments_text
```

Comment:
We use the space retrieved with each page to get the name of the space instead of the key/id because that is more semantically useful for searching